modules/disabling-applications-connected-to-open-data-hub.adoc

```diff
@@ -36,9 +36,9 @@ endif::[]
 ifndef::upstream[]
 --
 ifndef::self-managed[]
-* You have logged in to the {openshift-platform} web console.
-* You are part of the `cluster-admins` user group in {openshift-platform}.
-* You have installed or configured the service on your {openshift-platform} cluster.
+* You have logged in to the OpenShift web console.
+* You are part of the `cluster-admins` user group in your OpenShift cluster.
```

Comment:
There is also the `dedicated-admins` group on OSD, so we should mention it here. I a previous PR, I remember us discussing wording to note that `dedicated-admins` applies only to OSD.

Reply:
Addressed in latest commit.